src/server/server.py

```diff
@@ -29,76 +30,13 @@
 
 
 class Server(threading.Thread):
-    """Serves one or more bloom layers for inference, forward and backward; announces oneself to the DHT"""
+    """
+    Runs ModuleContainer, periodically checks that the network is balanced,
```

Comment:
[nit] ModuleContainer sounds somewhat too generic for a class that announces modules to DHT, accepts requests from p2pd, etc

Maybe
* Server -> LoadBalancedServer 
* ModuleContainer -> Server
?

Reply:
I agree that the name is not perfect but I didn't come up with better options. If we find one, we can rename this class later.